test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2550,7 +2499,7 @@ static public void testCheckCastValueTypeOnInvalidLtype() throws Throwable {
 	/*
 	 * Ensure that casting null to invalid Qtype class will throw a NoClassDef 
 	 */
-	@Test(priority=1, expectedExceptions=NoClassDefFoundError.class)
+	@Test(priority=1, expectedExceptions=NullPointerException.class)
```

Comment:
@tajila Heads up this test is still throwing NoClassDefFoundError, so is failing.

Reply:
testDefaultValueWithNonValueType is also failing due to OpenJ9 and Hotspot expecting different signature to defaultvalue(). Lmk how to proceed on both of them.

Reply:
You can disable this one. For each disabled test please open an issue describing what the RI behaviour is and how we differ. Also add me to it.